webauthn-rs-core depends on base64urlsafedata 0.1, actually requires 0.1.2

I upgraded `webauthn-rs` to 0.4.8 and my build broke:

```
error[E0277]: the trait bound `Base64UrlSafeData: Ord` is not satisfied
    --> /Users/coda/.cargo/registry/src/github.com-1ecc6299db9ec823/webauthn-rs-core-0.4.8/src/attestation.rs:1334:9
     |
1334 | /         ca_list
1335 | |             .cas
     | |________________^ the trait `Ord` is not implemented for `Base64UrlSafeData`
1336 |               .get(dgst.as_ref())
     |                --- required by a bound introduced by this call
     |
note: required by a bound in `BTreeMap::<K, V, A>::get`
    --> /Users/coda/.rustup/toolchains/stable-aarch64-apple-darwin/lib/rustlib/src/rust/library/alloc/src/collections/btree/map.rs:660:24
     |
660  |         K: Borrow<Q> + Ord,
     |                        ^^^ required by this bound in `BTreeMap::<K, V, A>::get`
```

I solved it by upgrading `base64urlsafedata` in my `Cargo.lock` to 0.1.2, but the version bound in `webauthn-rs-core` should be 0.1.2 and not 0.1.

--- a/webauthn-rs-core/Cargo.toml
+++ b/webauthn-rs-core/Cargo.toml
@@ -16,7 +16,7 @@ insecure-rs1 = []
 default = []
 
 [dependencies]
-base64urlsafedata = { version = "0.1", path = "../base64urlsafedata" }
+base64urlsafedata = { version = "0.1.2", path = "../base64urlsafedata" }
 webauthn-rs-proto = { version = "0.4.8", path = "../webauthn-rs-proto" }
 serde = { version = "1", features = ["derive"] }
 serde_cbor = { version = "0.12.0-dev", package = "serde_cbor_2" }

